src/main/resources/application.properties

```diff
@@ -34,7 +34,7 @@ spring.kafka.bootstrap-servers=${KAFKA_BROKERS_URL}
 app.kafka.brokersUrl=${KAFKA_BROKERS_URL}
 app.kafka.endringPaAvsluttOppfolgingTopic=${ENDRING_PA_AVSLUTT_OPPFOLGING_TOPIC}
 app.kafka.endringPaOppfolgingsBrukerTopic=${ENDRING_PA_OPPFOLGINGS_BRUKER_TOPIC}
-app.kafka.vedtakSendtTopic=${VEDTAK_SENDT_TOPIC}
+app.kafka.vedtakSendtTopic=pto.vedtak-sendt-v1
```

Comment:
@matiasvinjevoll Jeg glemte hva du sa/tenkte om dette, var det sånn at vi ikke hadde behov for å ta det som variabler i yaml-filene fordi topicene heter det samme i dev/prod uansett?

Reply:
@Mathiamu  de heter det samme i dev/prod...bare forskjell er kafka.pool verdi fra yaml file.